assets/js/containers/InteractiveBoard.js

```diff
@@ -1,20 +1,15 @@
 import { connect } from 'react-redux'
 import { GameEvents } from '../actions'
 import Board from '../components/Board'
-import { makeGetRequest, makePutRequest } from '../api'
+import { makePutRequest } from '../api'
 
 const mapStateToProps = state => ({
-  gameState: state.tictactoeFSM.gameState,
+  gameState: state.tictactoeFSM.round_state,
```

Comment:
I see we are mixing-and-matching snake_case and camelCase...
JavaScript is normally not snake_case

Reply:
Good point. 
I will fix to the moment of mapping of response from server in order to not impact Elixir code yet.

Reply:
Elixir is snake_case though :) Tradtionally, I have put a small layer into my clients that convert outside input in "any random case I don't control" into "nice case I like to write in my JS"

Reply:
True, here I'm trying to minimize number of layers. The trickiness of mapping layers I see in the moment of extensions of the entities: there is always a significant chance to forget to extend the mapping on some layer. In many cases old tests are not failing.

So far in this javascript the only mapping is happening in the FSM/Reducer, let's see if separate mapper will be required on coming functionality extension.